inference-engine/ie_bridges/java/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 Intel Corporation
+
+cmake_minimum_required(VERSION 3.0.0)
+
+find_package(Java REQUIRED)
+include(UseJava)
+
+set(JAVA_AWT_INCLUDE_PATH NotNeeded)
+
+project(inference_engine_java_api)
+
+if(DEFINED IE_MAIN_SOURCE_DIR)
+    set(InferenceEngine_LIBRARIES inference_engine)
+else()
+    find_package(InferenceEngineDeveloperPackage REQUIRED)
+    set(InferenceEngine_LIBRARIES IE::inference_engine)
+endif()
+
+find_package(JNI REQUIRED)
+
+file(GLOB_RECURSE sources cpp/*.cpp)
+add_library(${PROJECT_NAME} SHARED ${sources})
+
+file(GLOB_RECURSE java_source org/intel/openvino/*.java)
+add_jar(openvino_jar ${java_source} OUTPUT_NAME ${PROJECT_NAME} OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
+target_link_libraries(${PROJECT_NAME} PRIVATE ${InferenceEngine_LIBRARIES})
+target_include_directories(${PROJECT_NAME} PRIVATE ${JNI_INCLUDE_DIRS})
```

Comment:
can we move library part to `inference-engine/ie_bridges/java/cpp/CMakeLists.txt`?